BusinessModels/GeneSIS.md

```diff
@@ -132,7 +132,7 @@ TRL 8 at the end of the Project
     </td>
     <td rowspan="1">
       <p>
-Solution has been designed with specific support for all the infrastructure continuum. It is technology agnostic as well as cloud and platform independent. Faster time-to-market, better agility. It seamlessly integrates with security and privacy mechanisms for better trustworthiness.
+Solution has been designed with specific support for all the infrastructure continuum. It is technology agnostic as well as cloud and platform independent. Provide way to perform continuous deployment on local devices, reaching leafs of the infrastructure with no or limited access to Internet. Faster time-to-market, better agility. It seamlessly integrates with security and privacy mechanisms for better trustworthiness.
```

Comment:
Our solution has been designed with specific support for all the infrastructure continuum (IoT, Edge, and Cloud). It is technology agnostic as well as cloud and platform independent. The solution provides a way to perform continuous deployment on local devices, reaching leaves of the infrastructure with no or limited access to the Internet. Faster time-to-market, better agility. It seamlessly integrates with security and privacy mechanisms for better trustworthiness.